lib/wallaroo_labs/logging/wallaroo-logging.pony

```diff
@@ -0,0 +1,91 @@
+/*
+
+Copyright 2019 The Wallaroo Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ implied. See the License for the specific language governing
+ permissions and limitations under the License.
+
+*/
+
+// Tell ponyc that we're going to use the external C functions
+// in the libwallaroo-logging.a library.
+use "lib:wallaroo-logging"
+
+primitive Log
+  // severity levels
+  fun none(): U8   => U8(0)
```

Comment:
Not sure how much it matters, but in other logging tools I've seen, the levels typically increase
e.g. for python's logging it's:
```
0 - No logging
10 - debug
20 - info
30 - warning
40 - error
50 - critical
```
The use of multiples of 10 allows insertion of finer grained levels too (There's `21 - INFO2` in the testing harness,for example)

If it's not compatible with the implementation, feel free to ignore this comment.

Reply:
That's also similar Sendmail's debug tracing, if you squint a bit.  The hassle would be to give 'em all names.  20="info", 19="info-19" or "debug19"? or simply "19" or ""?

Sendmail's tracing doesn't bother with the debug/info/warning naming scheme.  It's purely numeric for both category & severity (which Sendmail would call "level").  So within a category like 0, there are various trace statements that use levels 1, 4, 10, 15, 20, 22, 40, 44, and 90.  (See https://docstore.mik.ua/orelly/networking/sendmail/ch37_05.htm).  If you enable tracing with `-d0.20`, then you'd see 20 and larger.

But that Sendmail thing is there only for debug tracing and not for the standard data logging that's sent to (for example) syslog.